src/codegen/luacg.cpp

```diff
@@ -1902,4 +1922,4 @@ void LuaTemplateParser::SetupModulePrefixes()
 
 	ADD_PREDEFINED_PREFIX( wxAC_DEFAULT_STYLE, wx. );
 	ADD_PREDEFINED_PREFIX( wxAC_NO_AUTORESIZE, wx. );
-}
+}
```

Comment:
Please keep a newline at the end of the file here.

Reply:
No worries, will do.